example/lib/widgets/buy_sheet.dart

```diff
@@ -232,6 +235,31 @@ class BuySheetState extends State<BuySheet> {
         collectPostalCode: true);
   }
 
+  Future<void> _onStartBuyerVerificationFlow() async {
+    var money = Money((b) => b
+      ..amount = 100
+      ..currencyCode = 'USD');
+
+    var contact = Contact((b) => b
+      ..givenName = "John"
+      ..familyName = "Doe"
+      ..addressLines = new BuiltList<String>(["London Eye", "Riverside Walk"]).toBuilder()
+      ..city = "London"
+      ..countryCode = "GB"
+      ..email = "johndoe@example.com"
+      ..phone = "8001234567"
+      ..postalCode = "SE1 7");
+
+    await InAppPayments.startBuyerVerificationFlow(
+        onCardOnFileBuyerVerificationSuccess: _onCardOnFileBuyerVerificationSuccess,
+        onBuyerVerificationFailure: _onBuyerVerificationFailure,
+        buyerAction: "Charge",
+        money: money,
+        squareLocationId: squareLocationId,
+        contact: contact,
+        paymentSourceId: "REPLACE_ME");
```

Comment:
Can we rename this as `REPLACE_WITH_PAYMENT_SOURCE_ID`?

Reply:
Yes.